subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidCachingSmokeTest.groovy

```diff
@@ -18,7 +18,9 @@ package org.gradle.smoketests
 
 import org.eclipse.jgit.api.Git
 import org.gradle.integtests.fixtures.executer.IntegrationTestBuildContext
+import spock.lang.Ignore
 
+@Ignore('contains cloning of other project')
```

Comment:
We then need to fix the other project I guess. Can you add a separate issue for this problem?